app/controllers/idv/jurisdiction_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Idv
+  class JurisdictionController < ApplicationController
+    include IdvSession
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_attempts_allowed
+    before_action :confirm_idv_needed
+    before_action :jurisdiction_form, only: %i[new create]
+
+    def new
+      analytics.track_event(Analytics::IDV_JURISDICTION_VISIT)
+    end
+
+    def create
+      result = @jurisdiction_form.submit(jurisdiction_params)
+      analytics.track_event(Analytics::IDV_JURISDICTION_FORM, result.to_h)
+      user_session[:jurisdiction] = @jurisdiction_form.state
```

Comment:
Nit, but how would you feel about naming this `:idv_jurisdiction`?

Reply:
i feel great about it!